src/Symfony/Component/Validator/composer.json

```diff
@@ -22,6 +22,8 @@
     },
     "require-dev": {
         "symfony/http-foundation": "~2.8|~3.0",
+        "symfony/http-kernel": "~2.8|~3.0",
+        "symfony/var-dumper": "~2.8|~3.0",
```

Comment:
@nicolas-grekas : Do you know what would be the proper version constraint to avoid [this issue](https://travis-ci.org/symfony/symfony/jobs/227378801#L3856)?